packages/ckeditor5-ui/src/bindings/injectcsstransitiondisabler.ts

```diff
@@ -76,7 +78,6 @@ export default function injectCssTransitionDisabler( view: View ): asserts view
 }
 
 export type ViewWithCssTransitionDisabler = View & {
```

Comment:
I'd move it to \`[csstransitiondisablermixin.ts](https://github.com/ckeditor/ckeditor5/pull/13222/files#diff-8eb907dbb7cc5b2bcfb06c5d79810a5efb1169160189e80f5220b334e04e4fef)\`